cryptopals/set1/4/4.txt

```diff
@@ -0,0 +1,327 @@
+0e3647e8592d35514a081243582536ed3de6734059001e3f535ce6271032
```

Comment:
Any particular use of this file? If someone wants to run the solution.py (ch4.py), he can download by himself. This feels redundant. Same goes for 7.txt and 8.txt in their respective directory.